### DISCUSSION TITLE
How do I mark a hwole series as watched quickly?

### DISCUSSION THREAD

**Discussion by zombiehoffa** [IgnisDa/ryot#614] [Q&A] [answered]
So far it seems like I have to add the series then go in and mark each season watched rather than being able to mark the entire thing as watched with one click somewhere. I am hoping I just missedit.

**Comment by IgnisDa** [comment, marked as answer]
Just click on the "Mark as seen" button for the last season. There should be a checkbox saying "Mark episodes before this as seen".
